edifact-edi/2.9/modules/ROOT/pages/edifact-edi-connector-reference.adoc

```diff
@@ -376,6 +376,15 @@ You can use this option with:
 ** CloudHub 1.0, CloudHub 2.0, and Runtime Fabric deployments with multiple workers or replicas
 ** Hybrid standalone deployments, including clustered Mule runtimes
 | AUTO |
+|Acknowledge every transaction | Boolean a| Whether to include the UCM Segment in the acknowledgement of every transaction:
```

Comment:
this isn't showing up in the preview for some reason, i think you may be missing some |